ultralytics/nn/autobackend.py

```diff
@@ -345,6 +345,7 @@ def wrap_frozen_graph(gd, inputs, outputs):
                     model_path=w,
                     experimental_delegates=[load_delegate(delegate, options={"device": device})],
                 )
+                device = "cpu"  # Required, otherwise PyTorch will try to use the wrong device
```

Comment:
Not terribly familiar with what devices are available for running `tflite` but might want to put a conditional here:

```python
if edgetpu:
    device = "cpu"  # Required, otherwise PyTorch will try to use the wrong device
```

Reply:
This isn't required since its already part of the edgetpu conditional above